tests/scripts/github-action-helper.sh

```diff
@@ -133,6 +133,7 @@ function validate_yaml() {
   cd cluster/examples/kubernetes/ceph
   kubectl create -f crds.yaml -f common.yaml
   # skipping folders and some yamls that are only for openshift.
+  # shellcheck disable=SC2010,SC2046 # Safe to grep with ls
   kubectl create $(ls -I scc.yaml -I "*-openshift.yaml" -I "*.sh" -I "*.py" -p | grep -v / | awk ' { print " -f " $1 } ') --dry-run
 }
```

Comment:
On a larger note, I don't expect a function named "validate" to create anything. Creation should be separate.

Another note, the comment does not explain why we are also ignoring quoting to prevent word splitting. Why?

What does `grep -v /` do?

In general, I would suggest find for this. You can select files, ignore files, only use a certain search depth, and also use `-exec` to get the `-f` args.

OR, you can process this in multiple lines to make it easier to interpret as a reader.

Reply:
I'll do it as a  separate commit in the end